storage/client_test.go

```diff
@@ -267,6 +271,26 @@ func (m *multiTestContext) replicateRange(raftID int64, sourceStoreIndex int, de
 	})
 }
 
+func (m *multiTestContext) unreplicateRange(raftID int64, source, dest int) {
+	rng, err := m.stores[source].GetRange(raftID)
+	if err != nil {
+		m.t.Fatal(err)
+	}
+
+	err = rng.ChangeReplicas(proto.REMOVE_REPLICA,
+		proto.Replica{
+			NodeID:  m.idents[dest].NodeID,
+			StoreID: m.idents[dest].StoreID,
+		})
+	if err != nil {
+		m.t.Fatal(err)
+	}
+
+	// Removing a range doesn't have any immediately-visible side
+	// effects, (and the removed node may be stopped) so return as soon
+	// as the removal has committed on the leader.
+}
```

Comment:
It would be trivial to have the removed range destroy its data:

    Range.Destroy()

Are you afraid to use this? I think we should hook it up.